src/wallet/rpcwallet.cpp

```diff
@@ -5134,11 +5135,12 @@ UniValue z_sendmany(const UniValue& params, bool fHelp)
     UniValue contextInfo = o;
 
     std::optional<uint256> orchardAnchor;
-    if (!ztxoSelector.SelectsSprout()) {
+    auto nAnchorDepth = std::min((unsigned int) nMinDepth, nAnchorConfirmations);
+    if (!ztxoSelector.SelectsSprout() && nAnchorDepth > 0) {
         // Allow Orchard recipients by setting an Orchard anchor.
         // TODO: Add an orchardAnchorHeight field to ZTXOSelector so we can ensure the
         // same anchor is used for witnesses of any selected Orchard note.
```

Comment:
Is there an issue for this TODO?

Reply:
This comment is actually irrelevant, I believe: we use a consistent anchor in the transaction builder, and we respect confirmation depth in note selection.